scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -1060,10 +1060,23 @@ def validate_assets(input_dirs: List[Path],
                 if check_environment_version:
                     error_count += validate_environment_version(asset_config)
 
-            if asset_config.type == assets.AssetType.PROMPT or asset_config.type == assets.AssetType.EVALUATIONRESULT:
-                error_count += validate_tags(asset_config, 'tag_values_shared.yaml')
+            if asset_config.type == assets.AssetType.EVALUATIONRESULT:
+                error_count += validate_tags(asset_config, 'evaluationresult/tag_values_shared.yaml')

```

Comment:
you might also want to consider adding tests as well (similar to https://github.com/Azure/azureml-assets/blob/main/test/test_validate_assets.py)